src/Components/Button/index.jsx

```diff
@@ -1,8 +1,11 @@
 import "./styles.css";
+
 export const Button = ({ label, handleOnClick, className }) => {
   return (
-    <button className={className} onClick={handleOnClick}>
-      {label}
-    </button>
+    <>
```

Comment:
You don't nee fragment component here, using button is enough since it's the single child

Reply:
```suggestion
```